### PR TITLE
Login and register

### PR DESCRIPTION
**Pull request by lan-nguyen91**

### CODE REVIEW

File: src/LoginSignup/LoginModal.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	render() {
+		return (
+			<div>
+				<main className="loginContainer">
+					<h1>Sign Up</h1>
+					<Formik
+						initialValues={{ email: '' }}
+						onSubmit={(values) => {
+							this.setState(() => ({
```
Comment (lan-nguyen91) [inline src/LoginSignup/LoginModal.js]: no need binding here, formik take care of it

File: src/LoginSignup/RegistrationAPI.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export async function registerEmail(userEmail) {
+	if (typeof userEmail !== 'string') console.warn('userEmail parameter is not a string.');
+	await axios
```
Comment (lan-nguyen91) [inline src/LoginSignup/RegistrationAPI.js]: let result = await fn() 

why .then if you are going to async await, if you expect error, wrap it in a try/catch

File: src/Misc/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function NotFound() {
+	return (
+		<div className="NotFound">
+			<section className="NotFound__section">
+				<p className="NotFound__section__p1">404</p>
+				<p className="NotFound__section__p2">PAGE NOT FOUND</p>
+        <p className="NotFound__section__p3">There's nothing here.</p>
```
Comment (lan-nguyen91) [inline src/Misc/NotFound.js]: indentation

File: src/Misc/NotFound.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function NotFound() {
```
Comment (lan-nguyen91) [inline src/Misc/NotFound.js]: export default = () =>   
// if you think this is cleaner

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import App from './App';
+
+ReactDOM.render(<App />, document.querySelector('#root'));
```
Comment (lan-nguyen91) [inline src/index.js]: let add hot reload here to make your life easier

https://medium.com/@brianhan/hot-reloading-cra-without-eject-b54af352c642

File: src/index.scss [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+.loginContainer {
+  @extend .flexCenter;
+  flex-direction: column;
+  border-radius: 7px;
+  height: 400px;
+  width: 520px;
+  margin: 13% auto 0;
+  background: $night1;
+
+  &__registerEmail-btn {
```
Comment (lan-nguyen91) [inline src/index.scss]: while this is all okay that you build custom CSS, and I will not stop you from doing it, there might be possibility I will throw this away, and replace with material-ui default btn

File: src/Misc/LandingPage.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import LoginModal from '../LoginSignup/LoginModal';
```
Comment (lan-nguyen91) [inline src/Misc/LandingPage.js]: let do absolute import to make your life easier
https://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d

File: src/LoginSignup/LoginModal.js (hunk 1 above)
Comment (do-kevin) [reply src/LoginSignup/LoginModal.js]: Are you talking about the `this.setState()`?

**Review by do-kevin — APPROVED** [review]

**Pull request merged**